README.md

```diff
@@ -163,7 +163,7 @@ Give a ⭐️ if you like this project!
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Thanks @ HAMMAD-SALEEM your very good advice I will try my best to follow it.